### PR TITLE
refactor: Add new Argo website

### PR DESCRIPTION
**Pull request by todaywasawesome**
This PR needs to be tested and we need to add the deploy components etc. 

One remaining issue is to fix the slack page.

### CODE REVIEW

**Comment by alexmt** [comment]
I think we don't have to store whole gsap package in source code. Only `DrawSVGPlugin.js` since it is not included in npm package. Tried to make this change and it works fine locally. Sent PR to your fork: https://github.com/todaywasawesome/argo-site/pull/1

**Comment by alexmt** [comment]
Updated CI to build site ( but not deploy ) on every PR: https://github.com/argoproj/argo-site/commit/12604509d59094d74eb0f66114f3003d4b92b828 . @todaywasawesome can you please rebase master to make sure CI can build new site version?

File: content/members/127labs/127labs.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....6042v3.9132h22.6602v-4.8233h-15.5618l8.2814-8.7819c3.3672-3.6402 5.8243-7.2349 5.8243-10.46558 0-4.86875-4.1407-8.00842-10.147-8.00842zm38.495 4.32273v-4.095218h-23.1607v9.282488h4.7323l.0455-4.45923h12.3766l-12.3766 27.02843h5.6878zm-21.6028 36.12507v-5.5935h-.7171v6.2748h3.4691v-.6813zm7.9601-.9053.6992 1.5866h.7798l-2.8057-6.2748h-.744l-2.8147 6.2748h.7619l.6992-1.5866zm-.3048-.6903h-2.8147l1.3984-3.1822zm2.7699-3.9979v6.2748h2.6892c1.4163 0 2.2589-.6364 2.2589-1.7032 0-.8605-.5289-1.4521-1.4342-1.5866.735-.1524 1.1563-.6813 1.1563-1.4163 0-.9861-.7619-1.5687-2.0258-1.5687zm.7171.6902h1.8914c.8426 0 1.3356.3586 1.3356.995 0 .6544-.493 1.0219-1.3356 1.0219h-1.8914zm0 2.6982h1.8914c1.0219 0 1.6135.4034 1.6135 1.0846 0 .7172-.5916 1.1205-1.6135 1.1205h-1.8914zm7.5656-3.4153c-1.2997 0-2.1693.6364-2.1693 1.6225 0 2.25 3.5319 1.3715 3.5229 3.0209 0 .6006-.5647.9681-1.4701.9681-.7082 0-1.506-.3227-2.0886-.8875l-.3227.6455c.5916.5647 1.488.9591 2.4023.9591 1.3446 0 2.2679-.6723 2.2679-1.7211.009-2.2769-3.5228-1.4342-3.5228-3.0478 0-.5288.5289-.8336 1.3356-.8336.502 0 1.1653.1524 1.7301.5289l.3048-.6723c-.52-.3496-1.264-.5827-1.9901-.5827z" fill="#643ddb" fill-rule="evenodd"/></g></svg>
```
Comment (alexmt) [inline content/members/127labs/127labs.svg:1]: Did you use some automation to generate members directory ? Can you please share script so we can use it in future? 


Here is what we have in current version: https://github.com/argoproj/argo-site/blob/master/scripts/logos.sh 
I periodically execute it manually to pull new members from USERS.md files . I'm pretty sure we can modify it to add folders for new members.

**Comment by alexmt** [comment]
Super awesome PR 🎉🎉🎉🎉! Thank you @todaywasawesome !

**Comment by alexmt** [comment]
>error gatsby@3.6.2: The engine "node" is incompatible with this module. Expected version ">=12.13.0". Got "10.24.1" 

Can you please bump node version in https://github.com/todaywasawesome/argo-site/blob/12604509d59094d74eb0f66114f3003d4b92b828/.github/workflows/gh-pages.yaml#L19

**Review by alexmt — APPROVED** [review]
LGTM.

I know we still need slack page but if this is the only blocker I would prefer to rollout new website and add it later. 

@todaywasawesome WDYT?

**Comment by todaywasawesome** [comment]
Agreed!

**Pull request merged**